example/full-config.php

```diff
@@ -112,11 +112,14 @@
                 'class' => \Doctrine\Common\Cache\ZendDataCache::class,
                 'namespace' => 'container-interop-doctrine',
             ],
+            'my_cache_provider' => [ 
+                'class' => \App\Doctrine\CustomCacheProvider::class, // its recommended to provide a factory to the container for the custom cache provider 
```

Comment:
The comment would be better to be like

> The class is looked up in the container

Or something... Don't make an assumption that they're using factories ;)

Reply:
👍